pulsar-broker/src/test/java/org/apache/pulsar/broker/stats/PrometheusMetricsTest.java

```diff
@@ -797,6 +804,10 @@ public void testBundlesMetrics() throws Exception {
 
         assertTrue(metrics.containsKey("pulsar_lb_bundles_split_total"));
 
+        assertTrue(metrics.containsKey("brk_lb_unload_latency"));
```

Comment:
I assume these latency metrics are automatically wired in the broker metric(without having to emit them explicitly).